app/pages/lab-pages-editor/icons/CloseIcon.jsx

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable react/react-in-jsx-scope */
+
+export default function CloseIcon() {
+  return (
+    <span className="icon fa fa-close" role="img" />
+  );
+}
```

Comment:
```suggestion
export default function CloseIcon({ ariaLabel = '' }) {
  return (
    <span className="icon fa fa-close" role="img" aria-label={ariaLabel} />
  );
}
```

Something like this might work.